extensions/2.0/Khronos/KHR_texture_astc/README.md

```diff
@@ -0,0 +1,236 @@
+<!--
+Copyright 2024 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR_texture_astc
+
+## Contributors
+
+- Wasim Abbas [@wasimabbas-arm](https://github.com/wasimabbas-arm)
+- Jan-Harald Fredriksen [@janharaldfredriksen-arm](https://github.com/janharaldfredriksen-arm)
+- Peter Harris [@solidpixel](https://github.com/solidpixel)
+- Mark Callow [@MarkCallow](https://github.com/MarkCallow)
+- Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)
+
+Copyright 2024 The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds the ability to specify textures using (ASTC compressed images)[https://github.com/ARM-software/astc-encoder/blob/main/Docs/FormatOverview.md] in KTX v2 containers. An implementation of this extension can use such images as an alternative to the PNG or JPEG images available in glTF 2.0 for more efficient asset transmission, reducing GPU memory footprint, higher texture throughput and faster execution.
```

Comment:
Incorrect link syntax: should be `[title](link)`.

Reply:
Good catch, Fixed https://github.com/KhronosGroup/glTF/pull/2369/commits/b79c60a407e3971d3f5dac4b8a9cfbf93b427834